scalableminds/webknossos
# Omit mag slider in TIFF export if only one mag is available

![image](https://github.com/user-attachments/assets/99e62d2c-8726-451d-808d-cb6c5324a29f)


### URL of deployed dev instance (used for testing):
- https://tiffexporthidesliderforonlyonemag.webknossos.xyz/

### Steps to test:
- Open an annotation with a layer that has more than one mag and a layer with only one mag
- Go to Menu > Download and open the TIFF download
- If you select a layer with one mag, there should not be a mag slider and only the current mag should be displayed
- If you select a layer with more than one mag, the mag slider should still be displayed
- Next to that, only annotations with actual additional coordinates should show a section about the chosen additional coords 

### TODOs:

### Issues:
- fixes #8002

------
(Please delete unneeded items, merge only when none are left open)
